foundry/foundry.py

```diff
@@ -804,23 +801,8 @@ def _load_data(self, file=None, source_id=None, globus=True, as_hdf5=False):
             return tmp_data
         else:
             raise NotImplementedError
-
-    
-    def toTorch(self, raw=None, split=None):
-        """Convert Foundry Dataset to a PyTorch Dataset
-
-        Arguments:
-            raw (dict): The output of running ``f.load_data(as_hdf5=False)``
-                    Recommended that this is left as ``None``
-                    **Default:** ``None``
-            split (string): Split to create PyTorch Dataset on.
-                    **Default:** ``None``
-
-        Returns: (FoundryDataset_Torch) PyTorch Dataset of all the data from the specified split
-
-        """
-        if not raw:
-            raw = self.load_data(as_hdf5=False)
+    def _get_inputs_targets(self, split: str = None):
+        raw = self.load_data(as_hdf5=False)
```

Comment:
Document this function:
- What does this do? 
- What does the "split" parameter mean"
- What does it return?

Reply:
Ok, fixed this.